### DISCUSSION TITLE
Email for Cosmos NoSQL RAG_Hack session

### DISCUSSION THREAD

**Discussion by Charlie-techLab** [microsoft/RAG_Hack#51] [General]
Hello!

I was recently in this call: https://www.youtube.com/watch?v=YZvNaYc_BLM&t=2100s and we were shared the following mail to send questions related to the RAG Hack: cdb4genai@microsoft.com but when I wanted to send a message I received an answer that says: Your message to cdb4genai@microsoft.com couldn't be delivered.
cdb4genai wasn't found at microsoft.com.

Please, can you clarify this?

Thank you 💯

**Comment by multispark** [comment]
@jcodella @jaydestro

**Reply by jcodella** [reply]
Hey @Charlie-techLab! Thanks for joining the session. There's a typo in the email, the correct one is:CDB4AI@microsoft.com. Sorry about that!

**Comment by Charlie-techLab** [comment]
Great, thank you a lot!
________________________________

**Comment by Charlie-techLab** [comment]
Hello, thank you for your answer!

I tried again to send an email but I received this message automatically:

Delivery has failed to these recipients or groups:

[CDB4AI@microsoft.com](mailto:CDB4AI@microsoft.com)
Your message wasn't delivered because the recipient's email provider rejected it.








Diagnostic information for administrators:

Generating server: IA3PR21MB4316.namprd21.prod.outlook.com
CDB4AI@microsoft.com
Remote server returned '554 5.7.0 < #5.7.133 smtp;550 5.7.133 RESOLVER.RST.SenderNotAuthenticatedForGroup; authentication required; Delivery restriction check failed because the sender was not authenticated when sending to this group>'
Original message headers:

ARC-Seal: i=2; a=rsa-sha256; s=arcselector10001; d=microsoft.com; cv=pass;
 b=ueQ6tdOl1Hxs8lVQlXfcJWV4kSPnfJRJBdnMVPOVosGnub3cMbkvcs2DQ/fqX+0f5855Zrxc/FtHE1lBSROicMMWZtDqhYwD5hf2gyf1S8y0Y9tU8QNXgnhZVkMO9ruc4lDFMYjVnFlZsOAkamHTCcBO1qVlOSuIlgrXr+JUeo0R107z5s0I9h30HremRKFJRGGbkIqWnKeIkA3702ITtbbozz6XR5o+EUlFphzrw40qRIJKjzXbu8N2RJitwxSTtUKQQmwnDLar1xHM8dIn5z8wmJ/x40MNbgug3iJMzq+EDN6+K4Ww5HB9vgFM/GK6nGt3eXhDKa8vpBBN/yKGQg==
ARC-Message-Signature: i=2; a=rsa-sha256; c=relaxed/relaxed; d=microsoft.com;
 s=arcselector10001;
 h=From:Date:Subject:Message-ID:Content-Type:MIME-Version:X-MS-Exchange-AntiSpam-MessageData-ChunkCount:X-MS-Exchange-AntiSpam-MessageData-0:X-MS-Exchange-AntiSpam-MessageData-1;
 bh=xYHAtE/uVEiEU6bs/iTZOHcGKIYnbizg6ghZ+NJua9E=;
 b=TWmfYpFJhZGp938nENkhzXH+vdgXhaVEfHDNeaqXrYTTJ0ERNVrg+pL78StPgTbDCKO6c0nvHFK5C6AACd1+m9bMCmdjvOprisGdvQWVZXRThRCnsF4OegbzWfI6lrJDL3JnEepD/NEgcrB7894CYYTA9gKEObCnv/IeAeqRCGNPm7KR4Ra4KhXHvXzvEOI31l1ITpfV+dj8fIVCeo9Ibw5bKpNMjEbS3H+quNVrAeTHVwAHUG9VSV41SDDubhzW3rxBXpL0BODspgiHYCmnD1rKVVPqdppDF213rIT8TsGPzd40/fFRZKQEsW95gQD7jz0l90FKHBBFvjw21nwUnw==
ARC-Authentication-Results: i=2; mx.microsoft.com 1; spf=pass (sender ip is
 40.107.92.112) smtp.rcpttodomain=microsoft.com
 smtp.mailfrom=studentambassadors.com; dmarc=bestguesspass action=none
 header.from=studentambassadors.com; dkim=pass (signature was verified)
 header.d=stdntpartners.onmicrosoft.com; arc=pass (0 oda=1 ltdi=1
 spf=[1,1,smtp.mailfrom=studentambassadors.com]
 dkim=[1,1,header.d=studentambassadors.com]
 dmarc=[1,1,header.from=studentambassadors.com])
Received: from DM6PR00CA0011.namprd00.prod.outlook.com (2603:10b6:5:114::24)
 by IA3PR21MB4316.namprd21.prod.outlook.com (2603:10b6:208:51d::17) with
 Microsoft SMTP Server (version=TLS1_2,
 cipher=TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384) id 15.20.7962.6; Thu, 5 Sep
 2024 15:21:45 +0000
Received: from MWH0EPF0009FF16.namprd00.prod.outlook.com
 (2603:10b6:5:114:cafe::99) by DM6PR00CA0011.outlook.office365.com
 (2603:10b6:5:114::24) with Microsoft SMTP Server (version=TLS1_2,
 cipher=TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384) id 15.20.7974.0 via Frontend
 Transport; Thu, 5 Sep 2024 15:21:44 +0000
Authentication-Results: spf=pass (sender IP is 40.107.92.112)
 smtp.mailfrom=studentambassadors.com; dkim=pass (signature was verified)
 header.d=stdntpartners.onmicrosoft.com;dmarc=bestguesspass action=none
 header.from=studentambassadors.com;
Received-SPF: Pass (protection.outlook.com: domain of studentambassadors.com
 designates 40.107.92.112 as permitted sender)
 receiver=protection.outlook.com; client-ip=40.107.92.112;
 helo=NAM10-BN7-obe.outbound.protection.outlook.com; pr=C
Received: from NAM10-BN7-obe.outbound.protection.outlook.com (40.107.92.112)
 by MWH0EPF0009FF16.mail.protection.outlook.com (10.167.242.6) with Microsoft
 SMTP Server (version=TLS1_2, cipher=TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384) id
 15.20.7980.0 via Frontend Transport; Thu, 5 Sep 2024 15:21:44 +0000
ARC-Seal: i=1; a=rsa-sha256; s=arcselector10001; d=microsoft.com; cv=none;
 b=j/asQL3obT5nrUA/3H6Li55hJw1XWVSjZJO2wfIRoLzAgcyhIna6p7mdno4/8B5iyacTNvyc4RGN1dCCTAf+5q3Oqs+/qJ45vTQy3H8jGYYL9MmEJUdzrHMG3IX4YXYPXlVid90HUKcJERN1Bz+5pZiPy6NhdD9laseOEcpleygr72IdMDpk6bTZvdkhCLcwtUxRJ6emSkGHjW8Nsj1/N72pl8WlTUyT7JafkNEhCY9qJKMhlmdBXC2toEprrUZuxT/g/nQXGL3FzMczM0WL6MFprSIz/VDm9Vglkfg05bExYf7HJjR7LHatQ23O32lsdFV0Ozoddo9tHqcKqqWzTA==
ARC-Message-Signature: i=1; a=rsa-sha256; c=relaxed/relaxed; d=microsoft.com;
 s=arcselector10001;
 h=From:Date:Subject:Message-ID:Content-Type:MIME-Version:X-MS-Exchange-AntiSpam-MessageData-ChunkCount:X-MS-Exchange-AntiSpam-MessageData-0:X-MS-Exchange-AntiSpam-MessageData-1;
 bh=xYHAtE/uVEiEU6bs/iTZOHcGKIYnbizg6ghZ+NJua9E=;
 b=TVD8cVbs0AY9tUaMNxfqEPOurEvZhDwixrPkEnM0zZefk+gUZ5Nq+5R8ccs/ylL+aZXCXMIQGPClcowCpRJnvnCLT023rJzPTT27yc9wKg5BonXzDwOmEGcRZo/8xmvEQ6yDWuQA/eh7qAvneybboE3v2xH4UJHzBuHi8NSzRxDTN3m67LQcqVKSZVYMipbd0PCFbvC3wmX0JL0Rg9D5z2SEkUkFpNuzpkXueq30PNo4WCC9+k4CTJQuyfRU+uw5o7m1asXCJHcCNqSAieW+T7/O8g/yPFxnYzXnlemJo43l4ZgHP4JcwzG72Rv6DEcSK2jUdYeJQpCGOaSCm22gnQ==
ARC-Authentication-Results: i=1; mx.microsoft.com 1; spf=pass
 smtp.mailfrom=studentambassadors.com; dmarc=pass action=none
 header.from=studentambassadors.com; dkim=pass
 header.d=studentambassadors.com; arc=none
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
 d=stdntpartners.onmicrosoft.com; s=selector2-stdntpartners-onmicrosoft-com;
 h=From:Date:Subject:Message-ID:Content-Type:MIME-Version:X-MS-Exchange-SenderADCheck;
 bh=xYHAtE/uVEiEU6bs/iTZOHcGKIYnbizg6ghZ+NJua9E=;
 b=h9abXnZfL6Z3ZS3zZO7HP8Dn4qn+d3lcMo02/i4Hv0yJho1nER91vn9ZCkVDFijR9lj6dOR9Ov9l4J4QPyOYXAVNETl++0mjhgidmw1d7e+DKRVSQ35idbx5VeGvQ2eAjVEpsNoib4DP/0+0EsJhKr1QHrIPbjhW9rtJo7c6DOE=
Received: from SJ0PR22MB4380.namprd22.prod.outlook.com (2603:10b6:a03:51a::10)
 by LV3PR22MB5870.namprd22.prod.outlook.com (2603:10b6:408:277::20) with
 Microsoft SMTP Server (version=TLS1_2,
 cipher=TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384) id 15.20.7918.27; Thu, 5 Sep
 2024 15:21:42 +0000
Received: from SJ0PR22MB4380.namprd22.prod.outlook.com
 ([fe80::cf8e:bbde:41f4:f5ea]) by SJ0PR22MB4380.namprd22.prod.outlook.com
 ([fe80::cf8e:bbde:41f4:f5ea%3]) with mapi id 15.20.7918.024; Thu, 5 Sep 2024
 15:21:42 +0000

**Comment by jcodella** [comment]
It should work now! I just tested it and verified.

Thanks for your patience! 😄

**Comment by Charlie-techLab** [comment]
Ok, thank you! 😄
